master/buildbot/changes/base.py

```diff
@@ -52,10 +52,13 @@ def _unclaimService(self):
 
 class PollingChangeSource(ChangeSource):
 
-    def __init__(self, name=None, pollInterval=60 * 10, pollAtLaunch=False):
-        ChangeSource.__init__(self, name=name)
-        self.pollInterval = pollInterval
-        self.pollAtLaunch = pollAtLaunch
+    def __init__(self, *args, **kwargs):
+        # Provide backwards compatibility
```

Comment:
I am not sure we really need this.

Anyway we eventually need to switch the change source to checkConfig/reconfigService